task_12/src/scss/index.scss

```diff
@@ -0,0 +1,195 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway:300,400');
+body {
+  background-color: #000000;
+  color: #ffffff;
+  font-family: 'Raleway', sans-serif;
+}
+
+.header {
+  &__wrapper {
```

Comment:
It's great you know SASS features. Please, use it wisely.

Imagine, somebody tries to improve your code, in the dev tools he found `.header_wrapper` and need to find the same class in the code. But _Ctrl+F_ does not work!

Use nesting only when it is not harmful.

Reply:
I removed nesting from this file. And, as I understand, it's better not to use this feature in almost all situations?

Reply:
SASS nesting is useful when you need to group an entity. For example, this is okay: 
```scss
a {
  color: blue;

  &:hover,
  &:focus {
    color: violet;
  }
}
```

It's obvious and good because `:hover` and `:focus` are completely connected with original selector and cannot exit without.

The basic idea is __group your selectors in one place__, in other words, have a single place for each selector.

In the example above we have just `a` and all related selectors to it in the single place.

I hope it's clear.

Reply:
Yea, now it clear, thank you.